src/Utils/sparseUtils.jl

```diff
@@ -1,9 +1,20 @@
 export sdiag
 
+import Base.kron
+export kron
+
 function sdiag(a::Vector)
 # S = sdiag(s) builds sparse diagonal matrix
 	n = length(a)
 	i = collect(1:n+1) # colptr
 	j = collect(1:n)   # rowval
 	return SparseMatrixCSC(n,n,i,j,a)
+end
+
+kron(v::SparseVector,A::SparseMatrixCSC) = kron(hcat(v...)',A)
```

Comment:
is there any reason why you are using `hcat(v...)` and not `SparseMatrixCSC(v)`. The further one is creating a dense vector and thus might be more inefficient. Haven't tested this and maybe there is a good reason to do it the way you did.